rasa/core/channels/socketio.py

```diff
@@ -144,6 +143,17 @@ def __init__(
         self.user_message_evt = user_message_evt
         self.namespace = namespace
         self.socketio_path = socketio_path
+        self.sio = None
+
+    def get_output_channel(self) -> Optional["OutputChannel"]:
+        if self.sio == None:
+            raise_warning(
+                "Output channel can't be"
+                "reproduced in case if sio equals to None"
+                "e.g. if the Rasa Server uses multiple Sanic workers"
```

Comment:
I'd rephrase it too be a bit less technical (currently you'd need to know the code to understand what `sio` means).

What do you think about this one?
```suggestion
                "SocketIO output channel cannot be recreated. "
                "This is expected behavior when using multiple Sanic "
                "workers or multiple Rasa Open Source instances. "
                "Please use a different channel for external events in these "
                "scenarios."
```

Reply:
This is much better 👍